includes/amp-helper-functions.php

```diff
@@ -702,9 +704,11 @@ function amp_get_content_sanitizers( $post = null ) {
 				'add_placeholder' => true,
 			),
 			'AMP_Gallery_Block_Sanitizer'     => array( // Note: Gallery block sanitizer must come after image sanitizers since itś logic is using the already sanitized images.
-				'carousel_required' => ! current_theme_supports( AMP_Theme_Support::SLUG ), // For back-compat.
+				'carousel_required' => ! is_array( $theme_support_args ), // For back-compat.
 			),
 			'AMP_Block_Sanitizer'             => array(), // Note: Block sanitizer must come after embed / media sanitizers since it's logic is using the already sanitized content.
+			'AMP_Nav_Menu_Toggle_Sanitizer'   => ! empty( $theme_support_args['nav_menu_toggle'] ) ? $theme_support_args['nav_menu_toggle'] : array(),
+			'AMP_Nav_Menu_Dropdown_Sanitizer' => ! empty( $theme_support_args['nav_menu_dropdown'] ) ? $theme_support_args['nav_menu_dropdown'] : array(),
```

Comment:
If theme support is not present, then these sanitizers actually should be omitted entirely. Right?

Reply:
Good catch!